[content] Case study: IBM commerce platform (#1929)
carbon-design-system/carbon-website: src/pages/case-studies/ibm-commerce-platform.mdx

addresses as users entered them, giving users confidence that they’d input the
correct address. Additionally, the support team no longer had to manually follow
up on invalid addresses, making the experience better for everyone.

## The approach

The transition to Carbon was an iterative process. Rather than updating the
entire UI at one time, the Commerce Platform team transitioned their product
slowly and tactfully. By adopting Carbon iteratively, they were able to see
which changes made substantial impacts and use that to inform their roadmap and
subsequent changes.

### Educate on the design system

Before jumping into adopting Carbon, it was important to spend time learning
about and getting comfortable with Carbon. Shixie Shi Trofimov, the design lead
of the transition project, led the developers through a crash course to align on
definitions and terminology. She wanted designers and developers to speak the
same language as they worked through a set of design specifications.

<Row>
<Column colLg={8}>

![Carbon token quiz](images/token-quiz.png)

<Caption>
  Carbon token quiz that helped get developers comfortable with Carbon token

>   naming conventions.

Reviewer: When do captions end in periods?

Reviewer 2: They end in periods when they're full sentences, but I'll double check the captions with Jan

  the order confirmation screen in production at the end of one sprint, and the
  order confirmation screen after in production at the end of the sprint plus
  two extra working days.
</Caption>

</Column>
</Row>

### Minimal QA

Once the team had been fully onboarded to Carbon, delivery required only minimal
design QA because they were using components straight out of the box.

### Continuity in experience

The project vastly improved the continuity in user experience with other IBM
touchpoints, including Support and My IBM. These experiences now feel continuous
and intuitive for users.

Within the Commerce Platform flow, pages are also unified and organized in the
same way, providing familiarity and a feeling of dependability.

<Row>
<Column colLg={8}>

![Continuity in experiences using Carbon for IBM.com.](images/continuity.png)

<Caption>
  Continuity in experiences using Carbon for IBM.com. From top to bottom:
  Product configuration, Checkout, Confirmation, and Profile.
</Caption>

</Column>
</Row>

## Positive outcomes

After the Carbon transition, more than 50% of customers are converting to
purchase, which is on par with industry benchmarks. The conversion rate was

> raised by 5% which has led to a large increase in orders and revenue.

Reviewer: This will land in the Summary? This 5% (or whatever the number needs to be per Andrew) is a big deal. I wonder if we need to emphasize on the impact of this number because at IBM's size, any increase in conversion rate is big.

Reviewer 2: Yep! The 5% and 50% numbers are both in the summary - I just pushed it if you want to take a look 🙂

## Next steps

The Commerce Team is currently working on improvements experience of their
configuration page. The next round of visual design improvements will focus on

> better aligning their interface with the Carbon for [IBM.com](http://ibm.com/)

Reviewer: Link to Carbon for IBM.com?

Reviewer 2: Good catch! Fixing now!

style models.

## Related resources

> - [IBM.com](http://ibm.com/) makers, go to

Reviewer: Do we want to use our resource tiles in here so the link to Carbon for IBM.com looks more official?

description:
  This case study examines the adoption of Carbon for the design and development
  of the new self-service purchase experience on IBM.com.
---

<PageDescription>

Learn about how the IBM.com team adopted Carbon for the design and development
of the new self-service purchase experience on IBM.com.

</PageDescription>

<AnchorLinks>

<AnchorLink>Summary</AnchorLink>
<AnchorLink>The challenge</AnchorLink>
<AnchorLink>Proposed solution</AnchorLink>
<AnchorLink>Our approach</AnchorLink>
<AnchorLink>Benefits</AnchorLink>
<AnchorLink>Positive outcomes</AnchorLink>
<AnchorLink>Next steps</AnchorLink>
<AnchorLink>Related resources</AnchorLink>

</AnchorLinks>

## Summary

The [IBM.com](http://ibm.com/) Commerce Platform team spent 3 three months
transitioning their checkout flow to use Carbon to improve their conversion rate
and overall product experience. They took an iterative approach,
starting first with small, isolated features to get comfortable designing and
developing with Carbon. Next they replaced existing elements with Carbon
components. Once all the components had been transitioned, the team focused on
updating page structure and layout.

> Since the Carbon refresh, the team’s delivery cycle has sped up and they spend

Reviewer 3: I think we should change the term "Carbon refresh" to "Carbon implementation" (it's used in several places in the text). "Refresh" implies (to me, at least) just a visual tweak.

Reviewer 2: Fixed!

> Since the Carbon refresh, the team’s delivery cycle has sped up and they spend

Reviewer 2: ```suggestion
Since the Carbon implementation, the team’s delivery cycle has sped up and they spend
```

introducing high-value features, and fixing existing bugs. An additional goal
was to reduce the amount of time IBM support was spending manually identifying
and resolving order issues.

### Analytics and research

Site analytics helped identify existing issues in the checkout process. There
were key issues to address during the transition to Carbon:

- There was a lack of visual and information hierarchy on the page.
- Critical information was missing when users encountered an error, and this
  blocked them from moving forward.
- Customers felt the checkout experience was unreliable, and couldn't quickly
  find what they wanted, so they abandoned the flow.

### Competitors

The Commerce team knew their eCommerce marketplace was falling behind
competitors, in both market share and user experience. Competitors had been
moving to a more modern experience that used data given by customers to
pre-populate fields, which significantly reduced the number of clicks. The team
used this project to strategically update their product to be able to better
compete in the marketplace.

## Proposed solution

### Carbon transition

> The Carbon refresh would make the interface more clear, meaning users wouldn’t

Reviewer 2: ```suggestion
The Carbon implementation would make the interface more clear, meaning users wouldn’t
```

### Iterate, iterate, iterate

Rather than updating all of the pages at once, the team took an iterative
approach. By starting with fresh features that integrated with the old UI, the
team could ship pieces iteratively.

The first stage involved switching individual components to their Carbon
version, starting with input fields then buttons. After transitioning the
individual components, the page structure and layout was updated.

With this approach, the Commerce team was able to refresh their product’s
experience, set the stage for larger future changes, and complete the project in
only three months.

<Row>
<Column colLg={12}>

![Checkout screen in January 2020, April 2020, and the future design](images/iterate.png)

<Caption>
  From left to right: checkout screen in January 2020 before the transition,
  April 2020 after the transition, and the future design.
</Caption>

</Column>
</Row>

## Benefits

There have been several benefits from the transition to Carbon for both the
Commerce team's own workflow and process, and the user experience. The team
especially found value in the impact of their delivery speed and need for QA.

### Faster delivery

> The entire Carbon refresh took only three months and since the transition, the

Reviewer 2: ```suggestion
The entire process for implementing Carbon took only three months and since the transition, the
```

  of the new self-service purchase experience on IBM.com.
---

<PageDescription>

Learn about how the IBM.com team adopted Carbon for the design and development
of the new self-service purchase experience on IBM.com.

</PageDescription>

<AnchorLinks>

<AnchorLink>Overview</AnchorLink>
<AnchorLink>The challenge</AnchorLink>
<AnchorLink>The proposed solution</AnchorLink>
<AnchorLink>The approach</AnchorLink>
<AnchorLink>Benefits</AnchorLink>
<AnchorLink>Positive outcomes</AnchorLink>
<AnchorLink>Next steps</AnchorLink>
<AnchorLink>Resources</AnchorLink>

</AnchorLinks>

## Overview

The IBM.com Commerce Platform team spent three months transitioning their
checkout flow to implement Carbon and improve their conversion rate and overall
product experience. They took an iterative approach, starting first with small,
isolated features to get comfortable designing and developing with Carbon. Next,
they replaced existing elements with Carbon components. Once all the components
had been transitioned, the team focused on updating page structures and layouts.

Since the Carbon implementation, the team’s delivery cycle has sped up and

> they spend less time on QA. Their product is now aligned with the

Reviewer: Any need to spell out quality assurance and not use the abbreviation QA? Same goes for UI below? Or do we think our audience will know what's up? I mean, we don't really say "quality assurance" or "user interface" in our day to day... so the abbreviation are probably fine. Just checking!

Reviewer 2: Good catch, Matt! We're going to write out the abbreviations. 🙂
